extensions/bodyparser-msgpack/README.md

```diff
@@ -0,0 +1,3 @@
+# @loopback/bodyparser-msgpack
+
+[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
```

Comment:
We will need to add some content to README, like basic instructions on how to use this extension. See [README guidelines](https://loopback.io/doc/en/contrib/README-guidelines.html).

I think it would be good to mention which msgpack implementation we are using under the hood, with a link to that library's npm or github page.